lib/AST/ASTImporter.cpp

```diff
@@ -1791,6 +1799,12 @@ static Error setTypedefNameForAnonDecl(TagDecl *From, TagDecl *To,
   return Error::success();
 }
 
+struct DefinitionCompleter {
```

Comment:
Could be defined in the scope of the `ASTNodeImporter::ImportDefinition` function.